projects/plugins/automattic-for-agencies-client/changelog/add-restful-features-available

```diff
@@ -0,0 +1,4 @@
+Significance: major
```

Comment:
Not really sure about the major here and all the plugins. It makes sense to have a major in the connection package, but I don't think it makes sense for the plugins

Reply:
What it really comes down to is "will this change potentially break things for people (properly) using the plugin?"

In this case, I'd say "probably not". I doubt the plugin exposes the functionality of this package method in its own UI or APIs. Anything using the package itself should itself be depending on the package rather than relying on any of our other plugins to provide it.

(I think we can ignore #17221 for the purposes of deciding if a plugin change is major).

Reply:
Many thanks for replying @anomiex. I will update the PR accordingly and change the significance of the changelogs in the plugins to minor then